projects/go-lib/src/lib/components/go-search/go-search.component.ts

```diff
@@ -61,19 +62,19 @@ export class GoSearchComponent implements OnInit {
     this.searchActive = true;
   }
 
-  leaveInput(event: any): void {
-    if (!this.elementRef.nativeElement.contains(event.relatedTarget)) {
+  leaveInput(event: FocusEvent): void {
+    if (event.relatedTarget && !this.elementRef.nativeElement.contains(event.relatedTarget)) {
```

Comment:
What is the case where a FocusEvent doesn't have a related target? I guess if you just click off of something?

Reply:
@AlexOverbeck exactly! This happens because the FocusEvent would be fired because a click off of the input, but not into another input. This causes an issue because `nativeElement.contains(null)` is true, apparently. Meaning that it would close the search if you clicked on the search results, which is not what we want. The other click event handler in this component handles closing it from a clicking off. This method should only handle the focus leave event.